README.md

```diff
@@ -79,8 +96,8 @@ import lindi
 # URL of the remote .nwb.lindi.json file
 url = 'https://kerchunk.neurosift.org/dandi/dandisets/000939/assets/11f512ba-5bcf-4230-a8cb-dc8d36db38cb/zarr.json'
```

Comment:
Could we update these example JSON to have the `.nwb.lindi.json` suffix?

Reply:
I'm in the process of batch renaming these in the bucket. It will take some time to iterate through all the dandisets. Then I will make the changes.

Reply:
This has been done in https://github.com/NeurodataWithoutBorders/lindi/pull/64